client/my-sites/domains/domain-management/transfer/transfer-out/transfer-lock.jsx

```diff
@@ -0,0 +1,50 @@
+/**
+ *  External Dependencies
+ *
+ * @format
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import SectionHeader from 'components/section-header';
+import { UPDATE_CONTACT_INFORMATION_EMAIL_OR_NAME_CHANGES } from 'lib/url/support';
+import { getSelectedDomain } from 'lib/domains';
+
+const TransferLock = props => {
+	const { translate } = props;
+	const { transferAwayEligibleAtMoment } = getSelectedDomain( props );
+
+	return (
+		<div>
+			<SectionHeader label={ translate( 'Transfer Domain' ) } />
+			<Card className="transfer-card">
+				{ translate(
+					'There is a transfer lock on this domain that expires at {{strong}}%(date)s.{{/strong}} ' +
```

Comment:
This is probably fine as it is, but the wording felt a little stilted to me. Maybe something like this might flow a little better: "Due to recent contact information updates, this domain has a transfer lock that expires at..."

Reply:
Updated, ty!